tools/internal_ci/linux/grpc_distribtests_php.sh

```diff
@@ -33,7 +33,7 @@ cp -r artifacts/* input_artifacts/ || true
 rm -rf artifacts_from_build_artifacts_step
 mv artifacts artifacts_from_build_artifacts_step || true
 
-# This step mostly just copies artifacts from input_artifacts (but it also does some wheel stripping)
+# This step simply collects php artifacts from subdirectories of input_artifacts/ and copies them to artifacts/
 tools/run_tests/task_runner.py -f package linux php -x build_packages/sponge_log.xml || FAILED="true"
```

Comment:
kind of, for some languages the package phase is just "collect the artifacts from subdirectories" (so that the per-language release processes can expect the packages being found in the right place).
For some languages (like C# and ruby's grpc-tools), we will still need a second phase ("build packages") but that's more of an exception.